src/Nordea/Components/DropdownFilter.elm

```diff
@@ -233,6 +247,21 @@ view attrs (DropdownFilter config) =
                         ]
                     ]
 
+              else if config.hasSearchIcon then
+                Icon.search
+                    [ css
+                        [ width (rem 1)
+                        , height (rem 1)
+                        , opacity (num 0.5)
+                        , position absolute
+                        , color Colors.blueNordea
+                        , left (rem 0.7)
+                        , top (pct 50)
+                        , transform (translateY (pct -50))
+                        , pointerEvents none
+                        ]
+                    ]
+
```

Comment:
It is not needed if you use `|> TextInput.withSearchIcon config.hasSearchIcon` above